app-service-deploy/main.js

```diff
@@ -0,0 +1,53 @@
+var minimist = require('minimist');
+var kuduService = require('./azure-arm-app-service-kudu').Kudu;
+
+let mopts = {

```

Comment:
can you use more descriptive name like argumentNames or argumentKeys or something here ?

Reply:
yes.. done